server/verify_test.go

```diff
@@ -461,6 +461,22 @@ func TestVerifyAttestationWithCerts(t *testing.T) {
 	}
 }
 
+func TestVerifyAutomaticallyUsesIntermediatesInAttestation(t *testing.T) {
```

Comment:
let's also add a test with overlapping `opts.IntermediateCerts` and `att.IntermediateCerts`